apps/ccd/ccd-next-hearing-date-updater/demo/00.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: ccd-next-hearing-date-updater
 spec:
   values:
+    java:
+      image: hmctspublic.azurecr.io/ccd/next-hearing-date-updater:pr-97-95f2a36-20221122154619 #{"$imagepolicy": "flux-system:demo-ccd-next-hearing-date-updater"}
```

Comment:
This line should be moved under `job` instead of creating a new `java` (and same for the other file) 🙏